e2e/types.ts

```diff
@@ -0,0 +1,22 @@
+export type E2EMessage = HandshakeMessage | RenderRequest | RenderResponse;
+
+export interface HandshakeMessage {
+  type: 'handshake';
+  data: {
+    os: string;
+    arch: 'paper' | 'fabric';
+    platformVersion: string;
+  };
+}
+
+export interface RenderRequest {
+  type: 'renderRequest';
+  data: any;
```

Comment:
Inside `data` there is just object which then is parsed to a component and rendered. I don't think we should touch this type as it literally reflects actual data inside there, it can be anything.

Reply:
Changed to `unknown`